code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -146,6 +125,41 @@
 					C.total_damage += detonation_damage
 				L.apply_damage(detonation_damage, BRUTE, blocked = def_check)
 
+/obj/item/kinetic_crusher/attack_secondary(atom/target, mob/living/user) //This is for aiming at yourself/adjacent mobs
```

Comment:
afterattack secondary works for melee range too cant see why this needs to exist